### ISSUE TITLE
Accessibility: Missing Alt Text for Images

### ISSUE THREAD

**Issue by Mackenzie-rae** [PavlAvstin/OZ-CSC-480-HCI-521-Fall-2023-#191]
There is missing text for all of these items listed
- 12 Movies under New Movies
- 12 Movies under Most Reviewed
- Hamburger Menu
- The x button(close out button)
- user pictures
- thumbs up and thumbs down buttons
